backend.native/tests/interop/objc/smoke.kt

```diff
@@ -213,6 +214,16 @@ fun createAndAbandonWeakRef(obj: NSObject) {
     WeakReference(obj)
 }
 
+fun testExceptions() {
```

Comment:
Do we have test for foreign exception?

Reply:
Not yet.

Reply:
Could we try to throw C++ exception in the test?

Reply:
Sure, but then it would crash the entire program.